presto-test-jdbc-compatibility-old-driver/src/test/java/io/prestosql/TestJdbcCompatibility.java

```diff
@@ -330,7 +330,7 @@ public void testSelectRow()
     {
         String query = "SELECT CAST(ROW(1, 2e0) AS ROW(x BIGINT, y DOUBLE))";
         checkRepresentation(query, JAVA_OBJECT, (rs, column) -> {
-            Map<String, Object> values = (Map<String, Object>) rs.getObject(column);
+            Map<String, Object> values = getRowAsMap(rs, column);
```

Comment:
I do not fully get this test class. How does it relate to `TestJdbcResultSetCompatibilityOldServer`?

Reply:
`TestJdbcCompatibility` tests all driver (using the old version of tests) against current server
`TestJdbcResultSetCompatibilityOldServer` tests current driver against old server

Reply:
Yeah - I meant `TestJdbcResultSetCompatibilityOldDriver` which also tests current server vs old driver

Reply:
Actually what you wrote about `TestJdbcCompatibilty`, is claimed to be done by `TJRSCOD`.
Quoting javadoc on that from `TestJdbcCompatibility`:
```
 * {@link TestJdbcResultSetCompatibilityOldDriver} in that regard is responsible for testing forward compatibility
 * as it's using old test code and old JDBC client against current server implementation.
```

Reply:
I am not sure i understand... do you want me to do a change here?

Reply:
No. Just thinking loudly.